src/components/Live/LiveProvider.js

```diff
@@ -77,9 +77,9 @@ export default class LiveProvider extends Component {
   render() {
     const {
       children,
-      className,
       code,
-      mountStylesheet,
+      language,
+      theme,
```

Comment:
Extreme nit: these went from being alphabetical to being slightly out of order. Feel free to ignore, just wanted to call it out